roles/mongodb/tasks/main2.yml

```diff
@@ -2,18 +2,19 @@
   include_tasks: install.yml
   when: mongodb_installed is undefined
 
-
-- name: Enable & Restart 'mongodb' systemd service if mongodb_enabled, incl daemon-reload (in case mongodb.service changed?)
+# daemon_reload is used to force systemd to reconize a newly installed .service file
+- name: Restart 'mongodb' systemd service if mongodb_enabled
   systemd:
     name: mongodb
     daemon_reload: yes
-    enabled: yes
+    enabled: no
```

Comment:
Why is MongoDB being disabled here?

(The `when: mongodb_enabled | bool` condition below implies the opposite.)

Reply:
moved the stanza